### PR TITLE
wlserver: Add support for mouse wheels

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

File: src/wlserver.cpp [hunk 1]
```diff
@@ -187,6 +187,15 @@ static void wlserver_handle_pointer_button(struct wl_listener *listener, void *d
 	wlr_seat_pointer_notify_frame( wlserver.wlr.seat );
 }
 
+static void wlserver_handle_pointer_axis(struct wl_listener *listener, void *data)
+{
+	struct wlserver_pointer *pointer = wl_container_of( listener, pointer, axis );
+	struct wlr_event_pointer_axis *event = (struct wlr_event_pointer_axis *) data;
+
+	wlr_seat_pointer_notify_axis( wlserver.wlr.seat, event->time_msec, event->orientation, event->delta, event->delta_discrete, event->source );
+	wlr_seat_pointer_notify_frame( wlserver.wlr.seat );
```
Comment (emersion) [inline src/wlserver.cpp:196]: Instead of sending `frame` manually, we should listen for `device->pointer->events.frame`.

Comment (emersion) [reply src/wlserver.cpp:196]: Ah, this version of wlroots might be too old for this. Nevermind!

Comment (emersion) [reply src/wlserver.cpp:196]: Hm, it seems like it's not too old after all!

Comment (valentindavid) [reply src/wlserver.cpp:196]: OK, I will try that.

Comment (valentindavid) [reply src/wlserver.cpp:198]: I have changed it.

**Review by emersion — APPROVED** [review]
LGTM, thanks!

File: src/wlserver.cpp (hunk 1 above)
Comment (emersion) [reply src/wlserver.cpp:198]: I didn't realize our `motion` event handlers weren't doing this properly.

**Pull request merged**